_data/venue.yml

```diff
@@ -5,6 +5,9 @@
 -
   name: "Dorien Huijser"
   image: UtrechtDorien.jpg
+  
```

Comment:
```suggestion
  -
  name: "Saskia and Steffen Bollmann"
  image: SaskiaBollmann.jpg
```

Reply:
Does that make sense?

Is this clear enough in the instructions?

The yml format can be very... Hot to put it? "picky"

Reply:
Yes, I just didn't see the '-' :P

In the instructions (5) and (6), it's says to add image filename to the yml file and then put a copy in the folder - with a specific filename. Maybe swap them around and first rename the image?

I'll commit the file, forgot to add it to git 😆